zc_common/fields.py

```diff
@@ -9,6 +10,10 @@
 US_PHONE_FORMAT = r'^(\+1\s?)?\(?(\d{3})\)?[\s-]?(\d{3})[\s-]?(\d{4})$'
 
 
+def numeric_uuid_generator():
+    return str(uuid.uuid4().int)[:10]
```

Comment:
Because this will be a default pk field for most if not all models going forward and we generally work with the kinds of data where a 10 digit id won't result in a clash on any reasonable time horizon. I think 38 character primary key for every model would be more annoying to work with than the benefits we would gain by having it.